security/mac-updates.md

```diff
@@ -0,0 +1,32 @@
+## Recommended Mac Updates
+
+## Capturing Status on the Command Line
+
+macOS ships with a tool called `softwareupdate` to scan, list, download, and
+install updates. If you're comfortable with typing in commands, you can see if
+recommended updates are available:
+
+  sudo softwareupdate -l
+
+The message _No updates are available._ will print at the end.
+
+## Capturing Status from the App Store
+
+You can check for updates from the App Store. From the Apple menu ()
+click on `App Store`, followed by `Updates` on the top row. You should see the
+message _No Updates Available_.
+
+![MacOS Updates in App Store](mac-updates.png)
+
+## Installing Updates on the Command Line
+
+You can install updates from the command line using the `softwareupdate` tool.
+To install recommended updates with progress indication run:
+
+  sudo softwareupdate -ivr
```

Comment:
It would appear that the `v` is not or is no longer supported.

```
[ruby-2.3.1p112]  ~ ツ sudo softwareupdate -ivr
softwareupdate: invalid option -- v
usage: softwareupdate <cmd> [<args> ...]

...

[ruby-2.3.1p112]  ~ ツ 
```